### DISCUSSION TITLE
Add comparison with twitter.js

### DISCUSSION THREAD

**Discussion by deadcoder0904** [plhery/node-twitter-api-v2#85] [Ideas]
I think https://github.com/twitterjs/twitter.js is an amazing library with TS support & well-maintained but it only supports v2.

Originally, I was going to use that library only but Twitter v2 API hasn't been fully on par with v1 API so I had to choose this one. This one is great too haha ;)

**Comment by alkihis** [comment]
Twitter.js is an object-oriented lib, use case clearly not the same ! :)
Plus, the table presented in README is only here for information purpose.
